src/compiler/scanner.ts

```diff
@@ -19,8 +19,12 @@ module ts {
         isReservedWord(): boolean;
         isUnterminated(): boolean;
         reScanGreaterToken(): SyntaxKind;
+        reScanLessThanSlashToken(): SyntaxKind;

```

Comment:
This is for ```</```.  So that would only happen when already in the JSX Element parsing case.  IN that case, you're already calling reScanJSXToken.  So this isn't needed as an explicit entrypoint (i don't think).